src/providers/messenger.js

```diff
@@ -0,0 +1,69 @@
+const domain = 'https://connect.facebook.net'
+
+const createCustomerchat = ({ 
+  providerKey,
+  themeColor = '',
+  loggedInGreeting = '',
+  loggedOutGreeting = '',
+  greetingDialogDisplay = '',
+  greetingDialogDelay = '',
+}) => {
+  if(!document.querySelector('.fb-customerchat')) {
+    const chat = window.document.createElement('div');
```

Comment:
Given this is a react plugin, and we've got the `Messenger` component, what do you think about moving this to be output to JSX inside `Messenger.js`?

This would involve creating a new hook to get the required props but I think it'd be a cleaner implementation.

Reply:
Please do it, your suggestion is great, I think you finish this would be better than me.